lib/src/Pos/Diffusion/Full.hs

```diff
@@ -289,8 +292,8 @@ diffusionLayerFull networkConfigOpts mEkgStore expectLogic =
 -- | Create kademlia, network-transport, and run the outbound queue's
 -- dequeue thread.
 runDiffusionLayerFull
-    :: forall d x .
-       ( DiffusionWorkMode d, MonadFix d )
+    :: forall d ctx x .
+       ( DiffusionWorkMode d, MonadSlotsData ctx d, MonadFix d )
```

Comment:
I've got to reject this. My goal is to keep all logic-layer related stuff out of constraints, and have it come from the `Logic d` value instead. The next slot duration can be derived from the mutable `BlockVersionData` on the `Logic d` interface instead.

Reply:
You're right, I'll fix it.